verification_tools/calc_limits.py

```diff
@@ -172,37 +172,37 @@ def calc_limits(configs, apertures, fluxes, scanfac=10, obsmode=None,
             inner_fac = 2.
             outer_fac = 5.
         else:
-            if obsmode['mode'] in ['msa','fixed_slit','lrsslit','lrsslitless','wfgrism','ssgrism','wfss','soss']:
+            if obsmode['mode'] in ['msa','fixed_slit','lrsslit','lrsslitless','wfgrism','ssgrism','wfss','soss','grism']:
                 inner_fac = 1.5
                 outer_fac = inner_fac+skyfac/2.
-            elif obsmode['mode'] in ['ifu','mrs','sw_imaging','lw_imaging','imaging','ami']:
+            elif obsmode['mode'] in ['ifu','mrs','sw_imaging','lw_imaging','imaging','ami','imager']:
```

Comment:
Do we need an else clause to avoid silent failures? (if mode = "girlscoutcookies" etc)

Reply:
We should. Added one here, and above.